frontend/src/components/dashboard/UserDataTable.vue

```diff
@@ -80,7 +85,11 @@ const filters = ref({
             <template #loading> Loading users data. Please wait. </template>
             <Column header="User name" sortable field="user.user_name">
                 <template #body="{ data }">
-                    <Icon icon="AvatarFilledIcon" medium />
+                    <Icon
+                        icon="user--avatar--filled"
+                        :size=IconSize.medium
+                        class="'custom-carbon-icon-user--avatar--filled'"
```

Comment:
do we need this class? I didn't see any override style for it?

Reply:
We don't. I missed this one, sorry about that